src/toil/job.py

```diff
@@ -982,9 +982,9 @@ def deleteLocalFile(self, fileStoreID):
                             os.remove(cachedFile)
                             cacheInfo.cached -= fileSize
                     self.logToMaster('Successfully deleted cached copy of file with ID '
-                                     '\'%s\'.' % fileStoreID)
+                                     '\'%s\'.' % fileStoreID, logging.DEBUG)
                 self.logToMaster('Successfully deleted local copies of file with ID '
-                                 '\'%s\'.' % fileStoreID)
+                                 '\'%s\'.' % fileStoreID, logging.DEBUG)
 
```

Comment:
It seem like there is a theme developing: we want invocations of file store methods made by Toil code to be logged at DEBUG, while user script invocations should be logged at INFO. We have @arkal's `quiet=` kwarg proposal. We could also use `inspect` to figure out whether the calling stack frame is for a function belonging to Toil. Other ideas?